optimus/df_transformer.py

```diff
@@ -749,63 +747,63 @@ def dateTransform(self, columns, currentFormat, outputFormat):
         :param  outputFormat    output date string format to be expected.
         """
         # Check if currentFormat argument a string datatype:
-        self.__assertTypeStr(currentFormat, "currentFormat")
+        self._assert_type_str(currentFormat, "currentFormat")
         # Check if outputFormat argument a string datatype:
-        self.__assertTypeStr(outputFormat, "outputFormat")
+        self._assert_type_str(outputFormat, "outputFormat")
         # Check if columns argument must be a string or list datatype:
-        self.__assertTypeStrOrList(columns, "columns")
+        self._assert_type_str_or_list(columns, "columns")
 
         if isinstance(columns, str):
             columns = [columns]
 
         # Check if columns to be process are in dataframe
-        self.__assertColsInDF(columnsProvided=columns, columnsDF=self.__df.columns)
+        self._assert_cols_in_df(columns_provided=columns, columns_df=self._df.columns)
 
         exprs = [date_format(unix_timestamp(c, currentFormat).cast("timestamp"), outputFormat).alias(
-            c) if c in columns else c for c in self.__df.columns]
+            c) if c in columns else c for c in self._df.columns]
 
-        self.__df = self.__df.select(*exprs)
+        self._df = self._df.select(*exprs)
 
-        self.__addTransformation()  # checkpoint in case
+        self._add_transformation()  # checkpoint in case
 
         return self
 
-    def ageCalculate(self, column, dateFormat, nameColAge):
+    def age_calculate(self, column, date_format, name_col_age):
```

Comment:
Hi @FavioVazquez , thank you for pointing this out. I am aware of this pyspark function but I did not foresee this to happen. I will update the parameter name.